apps/src/p5lab/utils.js

```diff
@@ -17,3 +17,33 @@ export function arrayEquals(a, b) {
 
   return true;
 }
+
+export function capitalizeFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+export function parsePathString(text) {
+  // Example string paths:
+  // 'sound://category_board_games/card_dealing_multiple.mp3'
+  // 'sound://default.mp3'
+  if (!text.startsWith('sound://') || !text.endsWith('.mp3')) {
```

Comment:
I think we have a constant for `'sound://'` in assetPrefix.js if you want to reuse that?